src/ruby/lib/grpc.rb

```diff
@@ -15,6 +15,7 @@
 ssl_roots_path = File.expand_path('../../../../etc/roots.pem', __FILE__)
 
 require_relative 'grpc/errors'
+require_relative 'grpc/structs'
```

Comment:
nit: requiring this is redundant?

Reply:
Possibly, but if other Structs are moved to Ruby then they should be loaded before `grpc_c`. I put the require here to be explicit about it.